src/rules/code-examples/unnecessaryElse.examples.ts

```diff
@@ -0,0 +1,76 @@
+/**
+ * @license
+ * Copyright 2019 Palantir Technologies, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as Lint from "../../index";
+
+// tslint:disable: object-literal-sort-keys
+export const codeExamples = [
+    {
+        description:
+            'Disallows "else" blocks following "if" blocks ending with "return", "break", "continue" or "throw" statement. ',
+        config: Lint.Utils.dedent`
+            "rules": { "unnecessary-else": true }
+        `,
+        pass: Lint.Utils.dedent`
+            if () {
```

Comment:
Coming back to these after a bit, the blank `()s` could be a little confusing for newcomers because they're invalid syntax in our code examples. Could you change it to `if (someCondition())` and similar?

Reply:
Done https://github.com/palantir/tslint/pull/4502/commits/a2842a9ad5ba17d0bb597296ad99b7bc0bbe5142